language/move-lang/tests/move_check/naming/duplicate_acquires_list_item.move

```diff
@@ -0,0 +1,8 @@
+module M {
+    resource struct R {}
+    resource struct X {}
+    foo() acquires R, X, R {
```

Comment:
What do you think of also having a test for multiple duplicate acquires items (e.g. `R, X, R, R`). And making sure this produces the errors expected.